modules/sms_email_reminder/cron_email_notification.php

```diff
@@ -7,11 +7,14 @@
 //
 // Created by:
 // Updated by:	Larry Lart on 10/03/2008
+// Updated by:  Daniel Pflieger on 7/16/2017 daniel@growlingflea.com growlingflea@gmail.com daniel@mi-squared.com
 ////////////////////////////////////////////////////////////////////
 
 // larry :: hack add for command line version
 $_SERVER['REQUEST_URI']=$_SERVER['PHP_SELF'];
 $_SERVER['SERVER_NAME']='localhost';
+$_SERVER['HTTP_HOST']='default';
+$ignoreAuth = true;
```

Comment:
basically anybody could do following to run this script in a web browser:
http://localhost/openemr/modules/sms_email_reminder/cron_email_notification.php

So, would put an exit; command at top of this script. And then place a comment that should move this script into a secure place and then remove the exit() command before using it.